service.subtitles.napisy24pl/service.py

```diff
@@ -306,6 +89,7 @@ def get_params():
 
     return param
 
+
 params = get_params()
```

Comment:
I just noticed that the stuff from here on out isn't encapsulated in anything.
Personally I'd prefer seeing this within a:
```python
if __name__ == '__main':
    ...
```
but isn't required to be changed.